ci-operator/step-registry/storage/create/csi-aws-efs/storage-create-csi-aws-efs-commands.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export STORAGEClASS_LOCATION=${SHARED_DIR}/efs-sc.yaml
```

Comment:
typo: lowercase `l` in STORAGEClASS_LOCATION

Reply:
actually there is a typo in efs operator too. I will keep this for now and I will fix it in a follow up.

Reply:
https://github.com/openshift/aws-efs-csi-driver-operator/blob/master/pkg/efscreate/create.go#L31

Reply:
fixed the typo in both places btw.